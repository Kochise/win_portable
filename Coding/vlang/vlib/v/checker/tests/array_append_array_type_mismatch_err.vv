fn main() {
	mut bc := []byte{}
	bc << [0xCA, 0xFE, 0xBA, 0xBE]
	println(bc)
}
