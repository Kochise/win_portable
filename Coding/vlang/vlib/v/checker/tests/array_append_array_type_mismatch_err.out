vlib/v/checker/tests/array_append_array_type_mismatch_err.vv:3:8: error: cannot append `[]int` to `[]byte`
    1 | fn main() {
    2 |     mut bc := []byte{}
    3 |     bc << [0xCA, 0xFE, 0xBA, 0xBE]
      |           ~~~~~~~~~~~~~~~~~~~~~~~~
    4 |     println(bc)
    5 | }
